tests/integration.rs

```diff
@@ -73,4 +193,164 @@ fn test_load_merc() {
             ..Default::default()
         }
     );
+
+    assert_eq!(
+        geotiff.model_extent(),
+        Rect::new(
+            Coord {
+                x: 1871032.9538880002,
+                y: 662408.6726400064
+            },
+            Coord {
+                x: 1901982.949391994,
+                y: 693358.6681440001
+            }
+        )
+    );
+}
+
+#[test]
+fn test_transform_by_model_tie_point_and_pixel_scale_pixel_is_area() {
+    test_transform(
+        "resources/austrian_capitals_model_tie_point_and_pixel_scale_pixel_is_area.tif",
+        RasterType::RasterPixelIsArea,
+    )
+}
+
+#[test]
+fn test_transform_by_model_tie_point_and_pixel_scale_pixel_is_point() {
+    test_transform(
+        "resources/austrian_capitals_model_tie_point_and_pixel_scale_pixel_is_point.tif",
+        RasterType::RasterPixelIsPoint,
+    )
+}
+
+#[test]
+fn test_transform_by_model_transformation_pixel_is_area() {
+    test_transform(
+        "resources/austrian_capitals_model_transformation_pixel_is_area.tif",
+        RasterType::RasterPixelIsArea,
+    )
+}
+
+#[test]
+fn test_transform_by_model_transformation_pixel_is_point() {
+    test_transform(
+        "resources/austrian_capitals_model_transformation_pixel_is_point.tif",
+        RasterType::RasterPixelIsPoint,
+    )
+}
+
+#[test]
+fn test_transform_by_model_tie_points_pixel_is_area() {
+    test_transform(
+        "resources/austrian_capitals_model_tie_points_pixel_is_area.tif",
+        RasterType::RasterPixelIsArea,
+    )
+}
+
+#[test]
+fn test_transform_by_model_tie_points_pixel_is_point() {
+    test_transform(
+        "resources/austrian_capitals_model_tie_points_pixel_is_point.tif",
+        RasterType::RasterPixelIsPoint,
+    )
+}
+
+fn test_transform<P: AsRef<Path>>(path: P, raster_type: RasterType) {
```

Comment:
Ok if we move the `test_transform_*` unit tests to a separate file? (I've also been thinking of renaming `tests/integration.rs` to something else, but can do that afterwards).

Reply:
`tests/transform.rs`?

Reply:
Looking at https://doc.rust-lang.org/book/ch11-03-test-organization.html#submodules-in-integration-tests, is it the convention to use `integration_` as the prefix for integration tests in Rust?

Reply:
No, I don't think so. The file is simply called ´integration_test.rs´ to highlight that it is an integration test as opposed to a unit test.